common/scripted_triggers/hfe_trigger.txt

```diff
@@ -69,10 +72,11 @@ hfe_trigger_is_awake = {
 
 hfe_trigger_is_assimilator = {
 	is_homicidal = no
-	or = {
+	OR = {
 		has_valid_civic = civic_machine_assimilator
+		has_valid_civic = hfe_civic_blight
```

Comment:
blight have civic_machine_assimilator too
redundant

Reply:
No harm, no foul. We simply agree to disagree on redundancy.

Reply:
No unnesesary calculations in mod

Reply:
You mean like the spawning of the Pharaoh without a determined gender, killing the Pharaoh if the Pharaoh is female, and then spawning the Pharaoh as a male?

You're one to talk.

Reply:
Still

Reply:
Okay, done.